packages/now-cli/src/util/output/create-output.ts

```diff
@@ -76,6 +77,20 @@ export default function createOutput({ debug: debugEnabled = false } = {}) {
     }
   }
 
+  function spinner(message: string, timeout: number = 300) {
+    if (debugEnabled) {
+      debug(`Spinner invoked (${message})`);
+      let isEnded = false;
+      return () => {
+        if (isEnded) return;
```

Comment:
I think it would be good to remove this condition so that we can see if end was invoked multiple times.

Also, I think you need to add the timeout

```js
setTimeout(() => debug('Spinner timed out after ${timeout} seconds.', timeout * 1000))
```

(assuming timeout is in seconds)

Reply:
`timeout` is the duration we wait *before* we show a loader (I agree the name is confusing!)

Reply:
I tried to avoid implementing something too complex, since we're also shipping it and it's only for debug. Adding the "timeout" to the "spinner invoked" log makes sense though 👍

Reply:
Oh so maybe `delay` is a better variable name?

Reply:
Sounds good!